www/components/pages/Meet/ActionItemCard/ActionItemCard.js

```diff
@@ -0,0 +1,159 @@
+import { TextField, Autocomplete, Button, Chip } from "@mui/material";
+import TaskAltIcon from "@mui/icons-material/TaskAlt";
+
+import actionItemStore from "../../../../store/features/action-item";
+import meetingStore from "../../../../store/features/meeting";
+
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import styles from "./ActionItemCard.module.scss";
+import shared from "../../../../styles/Shared.module.css";
+export default function ActionItemCard({
+  actionItem,
+  deleteUnsaved,
+  className,
+}) {
+  const dispatch = useDispatch();
+
+  const participants = useSelector((state) =>
+    meetingStore.selectors.participants(state, actionItem.meeting_id)
+  );
+
+  const [editing, setEditing] = useState(actionItem._id.includes("temp"));
+  const [name, setName] = useState(actionItem.name);
+  const [description, setDescription] = useState(actionItem.description);
+  const [assignedTo, setAssignedTo] = useState(actionItem.assigned_to);
+
+  const onSave = () => {
+    setEditing(false);
+
+    // When an action item is created, it receives a temp id until it is saved.
+    // Thus, if the action item has an id starting with temp, we need to create
+    // it.
+    if (actionItem._id.startsWith("temp")) {
+      dispatch(
+        actionItemStore.actions.create({
+          meeting_id: actionItem.meeting_id,
+          topic_id: actionItem.topic_id,
+          name,
+          description,
+          assigned_to: assignedTo,
+        })
+      );
+      deleteUnsaved(actionItem._id);
+    } else {
+      dispatch(
+        actionItemStore.actions.update({
+          _id: actionItem._id,
+          meeting_id: actionItem.meeting_id,
+          topic_id: actionItem.topic_id,
+          name,
+          description,
+          assigned_to: assignedTo,
+        })
+      );
+    }
+  };
+
+  const onDelete = () => {
+    // When an action item is created, it receives a temp id until it is saved.
+    // Thus, if the action item has an id starting with temp, we only need to
+    // delete it from the parent component's state.
+    if (actionItem._id.startsWith("temp")) {
+      deleteUnsaved(actionItem._id);
+    } else {
+      dispatch(actionItemStore.actions.delete(actionItem));
+    }
+  };
+
+  const maybeHandleShortcut = (e) => {
+    if (e.key === "Enter" && e.metaKey) {
```

Comment:
I couldnt get meta key to work and I dont have a mac but this worked for me:
```suggestion
    if (e.key === "Enter" && (e.metaKey || e.ctrlKey)) {
```